src/Translator/DonkiCme.php

```diff
@@ -140,6 +113,23 @@ public function views(): array  {
         return $base;
     }
 
+
+    /**
+     * Creates the short label used to describe this record.
+     */
+    public function shortLabel() {
+        $date = new DateTimeImmutable($this->data['startTime']);
+        $defaultLabel = $date->format('Y-m-d H:i:s');
```

Comment:
There's a date utility library you can use for this. The date library always uses `Y-m-d H:i:s` since that is almost standard throughout helioviewer (sometimes we do `Y-m-dTH:i:sZ`).

I think this file didn't use it since I hadn't written it yet.

```suggestion
use HelioviewerEventInterface\Util\Date;
// ...

$defaultLabel = Date::FormatString($this->data['startTime']); 
```